src/interactivity/win32/Clipboard.cpp

```diff
@@ -277,7 +277,10 @@ void Clipboard::CopyTextToSystemClipboard(const TextBuffer::TextAndColor& rows,
             int const iFontHeightPoints = fontData.GetUnscaledSize().Y * 72 / ServiceLocator::LocateGlobals().dpi;
             const COLORREF bgColor = ServiceLocator::LocateGlobals().getConsoleInformation().GetDefaultBackground();
 
-            std::string HTMLToPlaceOnClip = TextBuffer::GenHTML(rows, iFontHeightPoints, fontData.GetFaceName(), bgColor, "Windows Console Host");
+            // GH#5347 - Don't provide a title for the generated HTML, as many
+            // web applications will paste the title first, followed by the HTML
+            // content, which is unexpected.
+            std::string HTMLToPlaceOnClip = TextBuffer::GenHTML(rows, iFontHeightPoints, fontData.GetFaceName(), bgColor, "");

```

Comment:
should we go one step further and remove the parameter and the generation of the empty `<TITLE></TITLE>`?

Reply:
Yeah, if this is incorrect behavior, do that. That'll fix it for ConHost too then.

Reply:
It's done